### PR TITLE
Don't open default browsers. Log a clickable link instead

### PR DESCRIPTION
**Pull request by chewxy**
https://discuss.dgraph.io/t/slash-graphql-cli-opens-browsers-on-the-users-behalf-this-breaks-workflows/12925/2

### CODE REVIEW

**Review by gja — APPROVED** [review]

**Comment by chewxy** [comment]
@gja I can't merge this -  I have no merge rights.

**Comment by chewxy** [comment]
(p.s I also fixed a bug but I can't seem to push to this repo anymore)

**Comment by LockedThread** [comment]
This PR can be closed since they made a new shitty version of the login system. However, if they were to go back to the better version this addition would be greatly appreciated.

**Comment by danielmai** [comment]
I'm closing this PR since the slash-graphql CLI is deprecated in favor of the [Dgraph Cloud API](https://dgraph.io/docs/cloud/cloud-api/).

**Pull request closed without merging**
